wiki/Beatmap/Pattern/osu!/Bursts/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - bursts
+---
+
+# Bursts
+
+**Bursts** are a pattern where three or more [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on top of eachother in 1/4 or faster ticks.
+
```

Comment:
The transition between bursts and streams is fluid as far as my knowledge goes, but mentioning a general direction here might help improve the definition.

Something like:

There is no hard limit when bursts become streams but generally speaking, patterns with more than 10 consecutive hit objects are called streams.